making_change/making_change.py

```diff
@@ -3,8 +3,18 @@
 import sys
 
 def making_change(amount, denominations):
-  pass 
 
+	# if amount < 5 and amount >= 0:
+    # 	return 1
+  	# elif amount <0:
+    # 	return 0
+    # #recursive case
+  	# else:
+	# 	paths = 0 
+	# 	for c in denominations:
+	# 		paths =+ making_change(amount-c , denominations)
+	# 	return paths
```

Comment:
You're on the right path here, if you had to break it down a little further - how would you go about it?